website/docs/topics/data-collection.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Data collection
+---
+:::info
+
+At Unleash, we prioritize the privacy and security of our users' data. This document provides an overview of the data collected when running Unleash. We explain the purpose of data collection and provide instructions on managing data collection settings.
+
+:::
+
+## What data is collected {#what-data-is-collected}
+When running Unleash, we collect the following data:
+
+**Version and Instance ID**: A unique identifier and version for your Unleash instance. This ID allows us to track usage statistics and measure the adoption of Unleash across different installations and helps us ensure that you're using the latest version with the most up-to-date features and security enhancements.
+
+**Feature Usage Data**: Starting from **Unleash 5.3**, we collect additional data related to feature usage in Unleash.
+
+This includes the following data points:
+- The number of active feature toggles in use
+- The total number of users in the system
+- The total number of projects in the system
+- The number of custom context fields defined and in use
+- The number of user groups defined in the system
+- The number of custom roles defined in the system
+- The number of environments defined in the system
+- The number of segments in active use by feature toggles
+- The number of custom strategies defined and in use
+- Whether the instance uses SAML (Security Assertion Markup Language) for Single Sign-On
+- Whether the instance uses OIDC (OpenID Connect) for Single Sign-On
```

Comment:
I think we concluded that we don't collect this?

Reply:
Yuh, shot, yeeted